EsLint cnfiguration and package.json

teone opened the pull request:
Ciao @Granze, 
ho aggiunto le styleguide di Air BnB su EsLint, jsDoc e creato il file package.json, ho messo degli script di esempio:
- npm start
- npm test
- npm run doc

Sa manca qualcosa dimmi che aggiungo!

Commit:
Added Eslint configuration files

francomalatacca committed:
add JSHint config file

francomalatacca committed:
updated what's inside section of Readme.md

Commit:
Added AirBnB Styleguide enforced with eslint, index.js entry point, jsDoc and package.json

Commit:
Updated README and added start script

Commit:
Fixed npm test script

maur8ino commented:
@teone vai di rebase on milano-js:master :smile:

gcanti commented on README.md:
```diff
@@ -3,14 +3,31 @@
+
+## DOCUMENTATION
+
+Run `npm run doc` to generate the project documentation adn view it in the browser.
```
typo: "adn"

teone commented:
@maur8ino spero di non aver fatto casino con il rebase!

@gcanti grazie per il typo

Granze commented:
Ciao @teone la PR ha dei conflitti. Mi sa che il rebase non è andato benissimo. :)
Vedi se riesci a risolverli o se vuoi fare un'altra PR.

Commit:
Merge branch 'milanojs/master'

Conflicts:
	README.md

teone commented:
@Granze sistemati i conflitti :)

maur8ino commented:
:+1:

Granze commented:
Grande @teone, ultime due robe e poi non ti rompo più: 
- nei file eslint\* lascia una riga vuota alla fine
- leva jshintrc dal what's inside
- la parte con le linee guida sulla contribuzione la lascerei nel file separato

Grazie ;)

Commit:
Removed:
- removed .jshintrc
- added empty line in .eslintrc and .eslintignore
- moved contribution rules in CONTRIBUTING.md

teone commented:
@Granze done!

Ma tu non eri in vacanza? Cosa ci fai attaccato al PC?

Granze commented:
Grazie @teone!
No, non sono in ferie ma lavoro dalla casa in toscana che comunque è meglio di niente. ;)
